dalc/optimint_server.toml

```diff
@@ -0,0 +1,18 @@
+[base]
+  laddr = "0.0.0.0:4200"
```

Comment:
Indentation isn't needed for toml. I'd remove it just because I've literally never seen a toml file with indentation of non-multiline-fields before.

Reply:
Fixed.